rules/config

```diff
@@ -221,3 +221,12 @@ ENABLE_ASAN ?= n
 
 # reset default container registry from dockerhub to other
 DEFAULT_CONTAINER_REGISTRY ?=
+
+# ENABLE_FIPS_FEATURE - support FIPS feature, only for amd64 or arm64, armhf not supported yet
+# ENABLE_FIPS - support FIPS flag, if enabled, no additional config requred for the image to support FIPS
+ifeq ($(CONFIGURED_ARCH), armhf)
+ENABLE_FIPS_FEATURE ?= n
+else
+ENABLE_FIPS_FEATURE ?= y
+endif
+ENABLE_FIPS ?= n
```

Comment:
So `ENABLE_FIPS_FEATURE` affects build-time, and causes the FIPS binaries to get installed into the image. `ENABLE_FIPS` affects runtime, and tells the FIPS binaries to actually use FIPS. Is this correct?

If so, maybe renaming `ENABLE_FIPS` to `ENABLE_FIPS_AT_RUNTIME` may explain it better? Also, this flag would depend on `ENABLE_FIPS_FEATURE` to be `y`, so adding a check for that would be good.

Reply:
@saiarcot895 , thanks for your comment. Yes, the ENABLE_FIPS affects the runtime, it will enable the fips by default without any config. In the last community review, it is requested to add the feature. If the value is not set, we have to set change the Linux parameter in the sonic image installation or upgrade time.
Looks like ENABLE_FIPS_AT_RUNTIME is more clear, but a little strange, not necessary to emphasize it, how about using ENABLE_FIPS without change, but if you still think better to change to ENABLE_FIPS_AT_RUNTIME, I can change it.

For the ENABLE_FIPS check, I have added a check in Makefile.work, if the ENABLE_FIPS is set, but ENABLE_FIPS_FEATURE is not set, the build will be failed.

Reply:
Ok, I'm fine with leaving the names as-is.